Gaijinizer patch/Map042.txt

```diff
@@ -290,265 +317,293 @@ What!?#afrg
 
  とくべつな　エンプーサ　創られた。
 > CONTEXT: Map042/events/5/pages/1/95/Dialogue/enemy2
-Certainly for Me "Shelter" I and another one
-A special kind of empuse was created.#MTLed
+Certainly... there were... two Empuses...
+specially made... to attack... the shelter...#Ron
```

Comment:
We should make this consistent with the ruling on -195/+215 with what we wish to call "specially made".